iroha_client_cli/README.md

```diff
@@ -0,0 +1,81 @@
+# Iroha CLI Client
+
+Iroha CLI Client provides an ability to interact with Iroha Peers Web API without direct network usage.
+It's a "light" client which only converts Command Line Interface commands into Iroha Web API Network Requests.
+
+## Installation
+
+//TODO:
+
+## Examples
+
+Full description and list of commands detailed in `iroha_cli --help`.
+
+```
+$: ./iroha_client_cli --help
+Iroha CLI Client 0.1.0
```

Comment:
I think it would be great to generate man page, readme, etc. by CI so it will be always up to date.